pages/_app.tsx

```diff
@@ -7,30 +7,48 @@ import {
   InMemoryCache,
   ApolloProvider,
 } from "@apollo/client";
-import { SUBGRAPH_URL } from "../lib/constants";
-import {
-  Provider as MultiAuth,
-  // eslint-disable-next-line import/named
-  PartialConnectorConfig,
-} from "@ceramicstudio/multiauth";
+import { SUBGRAPH_URL, NETWORK_NAME } from "../lib/constants";
 import "../styles.scss";
-import {
-  injected,
-  walletconnect,
-  // fortmatic,
-  // portis,
-  torus,
-} from "../lib/wallets/connectors";
 import { AppProps } from "next/app";
 
-const connectors = new Array<PartialConnectorConfig>(
-  { key: "injected", connector: injected },
-  { key: "walletConnect", connector: walletconnect },
-  // { key: "fortmatic", connector: fortmatic },
-  // { key: "portis", connector: portis },
-  { key: "torus", connector: torus }
+import { chain, configureChains, createClient, WagmiConfig } from "wagmi";
+import { infuraProvider } from "wagmi/providers/infura";
+import {
+  connectorsForWallets,
+  RainbowKitProvider,
+  lightTheme,
+} from "@rainbow-me/rainbowkit";
+import {
+  injectedWallet,
+  metaMaskWallet,
+  walletConnectWallet,
+  ledgerWallet,
```

Comment:
Can we add Coinbase Wallet and Brave Wallet as the second to last and last wallet options?